float32 ball_on_frame_x
float32 ball_on_frame_y
float32 ball_on_frame_theta
float32 ball_on_frame_dist

float32 ball_on_field_x
float32 ball_on_field_y
float32 ball_on_field_theta
float32 ball_on_field_dist
uint8 ball_status

uint16[] obs_on_field
uint8[] obs_limit